src/postprocessors/NekSidePostprocessor.C

```diff
@@ -30,7 +30,8 @@ NekSidePostprocessor::validParams()
 }
 
 NekSidePostprocessor::NekSidePostprocessor(const InputParameters & parameters)
-  : NekPostprocessor(parameters), _boundary(getParam<std::vector<int>>("boundary"))
+  : NekPostprocessor(parameters),
+    _boundary(getParam<std::vector<int>>("boundary"))
```

Comment:
Why was this line needed?

Reply:
Sorry nothing should have been changed here, I'll fix this.